documentation/source/user_section/tutorials/shape-analysis/compute-csa-and-other-shape-metrics/other-shape-metrics.rst

```diff
@@ -3,11 +3,25 @@
 Other shape metrics
 ###################
 
-The csv files generated by ``sct_process_segmentation`` also include metrics to analyse the shape of the spinal cord in the axial plane, such as ellipticity, antero-posterior and right-left dimensions. These are of particular interest for studying cord compression. See `Martin et al. BMJ Open 2018 <https://bmjopen.bmj.com/content/8/4/e019809>`_ for an example application in degenerative cervical myelopathy.
+The csv files generated by ``sct_process_segmentation`` also include additional metrics (besides area) to analyse the shape of the spinal cord in the axial plane:
 
 .. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/shape-metric-computation/sct_process_segmentation-shape-metrics.png
    :align: center
 
+================  =======
+    CSV key       Meaning
+================  =======
+``diameter_AP``   Antero-posterior diameter (AP diameter)
+``diameter_RL``   Transverse diameter
```

Comment:
One final question: Is "Transverse" the right term here? I'm looking at this diagram, and:

![image](https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/16181459/bb4667e7-423b-4ab5-8180-49c21a68b085)

The "transverse" plane refers to the axial plane as a whole. So, maybe it would be clearer to just say "Right-left diameter" to match the AP line and keep things consistent?

```suggestion
``diameter_AP``   Antero-posterior diameter
``diameter_RL``   Right-left diameter
```

Reply:
Both `Right-left` and `Transverse` are correct. But more commonly used in publications is `Transverse diameter`. This is why we used "transverse" in https://neurolibre.org/papers/10.55458/neurolibre.00017.
But yes, to be consistent with `diameter_RL` provided by `sct_process_segmentation`, `Right-left` is fine. Thanks!

Reply:
i would also go with diameter_RL

Reply:
Thank you for the clarification! I learned something new today :)

Reply:
> i would also go with diameter_RL

sorry i meant: i would go with "Right-left diameter"-- it is much more intuitive-- not everybody is aware of the term transverse diameter (i was not)

Reply:
((I had made the change to Right-left diameter when I saw your first :+1: on my initial question, and this change has been merged already, so all good here!))